java/foundation/src/org/xtreemfs/foundation/pbrpc/generatedinterfaces/PingServiceClient.java

```diff
@@ -1,4 +1,4 @@
-//automatically generated from Ping.proto at Wed Nov 12 13:02:31 CET 2014
```

Comment:
Please skip generated files that only have a new timestamp in future (no need to revert this commit).